core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -43,6 +43,9 @@
   @BaseMessage("APPLY operator is not allowed under the current SQL conformance level")
   ExInst<CalciteException> applyNotAllowed();
 
+  @BaseMessage("JSON path expression must be specified after the JSON value expression")
+  ExInst<CalciteException> jsonPathAfterJsonValueExpression();
```

Comment:
Thank you for the modification, @XuQianJin-Stars. But the method name seems to be a little confusing. I suggest to change it to "jsonPathMustBeSpecified" or something similar.

Reply:
I will modify it immediately.

Reply:
hi @zhztheplayer I've changed the method name.